Zend/zend_globals.h

```diff
@@ -127,6 +127,8 @@ struct _zend_compiler_globals {
 	zval **static_members_table;
 	int last_static_member;
 #endif
+
+	zend_bool have_called_openlog;
```

Comment:
I don't understand why this need to in CG? PG maybe better? thanks

Reply:
Well, I asked and no one answered...

Reply:
Well, I asked and no one answered...

Isn't `core_globals` for configurable state?

Reply:
CG is for *compiler* globals, PG is more suitable in this case IMO.

Reply:
@pprindeville This really should be in PG...

Reply:
> @pprindeville This really should be in PG...

Where do you initialize a `core_globals` variable that's not user settable?